salt/metalk8s/kubeadm/init/kubelet-start/configured.sls

```diff
@@ -0,0 +1,62 @@
+{%- from "metalk8s/map.jinja" import kubelet with context %}
+
+Create kubelet service environment file:
+  file.managed:
+    - name: {{ kubelet.service.environment_file }}
+    - source: salt://metalk8s/kubeadm/init/kubelet-start/files/kubeadm.env
+    - template: jinja
+    - user: root
+    - group: root
+    - mode: 644
+    - context:
+        options: {{ kubelet.service.options }}
+
+{#- Default dataset for kubelet config #}
```

Comment:
Is it a comment? Can't be simply `# Default dataset for kubelet config`?

Reply:
No because it's a comment about jinja stuff so it's a jinja comment :)